### PR TITLE
Adds the Dollar Circle Illustration

### PR DESCRIPTION
**Pull request by melissaeckels**
Implements the `DollarCircleIllustration` to the component library.

* Observe it in Storybook under `atoms/illustrations`.

Used for personal loan things.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)

### CODE REVIEW

File: components/atoms/illustrations/DollarCircleIllustration/DollarCircleIllustration.js [hunk 1]
```diff
@@ -0,0 +1,666 @@
...
+      return (
+        <svg
+          {...elementProps}
+          id={`${id}-illuminated`}
+          viewBox="0 0 140 140"
+          xmlns="http://www.w3.org/2000/svg"
+          xmlnsClink="http://www.w3.org/1999/xlink"
+        >
+          <pattern
+            id="a"
```
Comment (labichn) [inline components/atoms/illustrations/DollarCircleIllustration/DollarCircleIllustration.js]: Should this be ``id={`${id}-a`}`` and its references below? And pattern `b` too, mutatis mutandis?

Comment (JamesIves) [reply components/atoms/illustrations/DollarCircleIllustration/DollarCircleIllustration.js]: yup, fixed

File: components/atoms/illustrations/DollarCircleIllustration/DollarCircleIllustration.js [hunk 2]
```diff
@@ -0,0 +1,666 @@
...
+            height="26.39"
+            rx="4.7"
+            transform="matrix(.98893833 .14832724 -.14832724 .98893833 9.53 -10.89)"
+            width="75.35"
+            x="40.1"
+            y="45.25"
+          />
+          <path
+            d="m112.34 50.29-65.23-9.78a4.7 4.7 0 0 0 -5.34 3.94l-2.52 16.81a4.69 4.69 0 0 0 4 5.34l65.23 9.78a4.7 4.7 0 0 0 5.34-3.94l2.52-16.81a4.69 4.69 0 0 0 -4-5.34z"
+            fill="url(#a)"
```
Comment (labichn) [inline components/atoms/illustrations/DollarCircleIllustration/DollarCircleIllustration.js]: These pattern ID references need to be updated too, right?

Comment (JamesIves) [reply components/atoms/illustrations/DollarCircleIllustration/DollarCircleIllustration.js]: wooooopzzzzzz

**Pull request merged**